Beta_Integrations/SentinelOne-V2/SentinelOne-V2.yml

```diff
@@ -0,0 +1,1264 @@
+    description: Move agents to a new group.
+    execution: false
+    name: sentinelone-move-agent
+    outputs:
+    - contextPath: SentinelOne.Agents.AgentsMoved
+      description: The number of agent that moved to another group
+      type: Number
+  - arguments:
+    - default: false
+      description: The ID of the group to delete
+      isArray: false
+      name: group_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Delete a group by the group ID
+    execution: false
+    name: sentinelone-delete-group
+  - arguments:
+    - default: false
+      description: 'The ID of the agent from which to retrive the processes from '
+      isArray: false
+      name: agents_ids
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieve running processes for a specific agent.
+    execution: false
+    name: sentinelone-agent-processes
+    outputs:
+    - contextPath: SentinelOne.Agents.memoryUsage
+      description: Memory usage (MB)
+      type: Number
+    - contextPath: SentinelOne.Agents.startTime
+      description: The process start time
+      type: Date
+    - contextPath: SentinelOne.Agents.pid
+      description: The process ID
+      type: Number
+    - contextPath: SentinelOne.Agents.processName
+      description: 'The name of the process '
+      type: String
+    - contextPath: SentinelOne.Agents.cpuUsage
+      description: CPU Usage (%)
+      type: Number
+    - contextPath: SentinelOne.Agents.executablePath
+      description: Executable path
+      type: String
+  - arguments:
+    - default: false
+      description: Events matching the query search term will be returned
+      isArray: false
+      name: query
+      required: true
+      secret: false
+    - default: false
+      description: Events from a specific date
+      isArray: false
+      name: from_date
+      required: true
+      secret: false
+    - default: false
+      description: Events to a specific date
+      isArray: false
+      name: to_date
+      required: true
+      secret: false
+    deprecated: false
+    description: Start a Deep Visibility Query and get the queryId, use queryId for
+      all other APIs such as Get All Events and Get Query Status
+    execution: false
+    name: sentinelone-create-query
+  - arguments:
+    - default: false
+      description: The query ID. Get query_id from create-query command
+      isArray: false
+      name: query_id
+      required: true
+      secret: false
+    - default: false
+      description: The limit of the results to be returned
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    deprecated: false
+    description: Get all Deep Visibility events from query
+    execution: false
+    name: sentinelone-get-events
+  dockerimage: demisto/python3:3.7.3.286
+  isfetch: true
+  runonce: false
+  script: '-'
+  type: python
+releaseNotes: New integration - SentinelOne V2
```

Comment:
we don't use releaseNotes in yml anymore, but use external file under release notes folder. Should see @yaakovi 's explanation on slack for more info.

Reply:
Also we should make sure we explain to users this is beta and if any known issue we know about in advance, to prevent confusion.